app/views/coronavirus_local_restrictions/show.html.erb

```diff
@@ -40,7 +36,14 @@
 
     <%= sanitize(t("coronavirus_local_restrictions.lookup.body_content")) %>
 
-    <%= form_tag find_coronavirus_local_restrictions_path, method: :get, class: "coronavirus-local-restrictions__postcode-form" do |f| %>
+    <%= form_tag find_coronavirus_local_restrictions_path,
+      method: :get,
+      class: "coronavirus-local-restrictions__postcode-form",
+      data: {
+        module: "coronavirus-local-restrictions-postcode-form",
+        "error-code": @search&.error_code,
+        "error-message": error_key ? t("coronavirus_local_restrictions.errors.#{error_key}.error_description") : nil,
```

Comment:
Should we put this logic on the top so we don't have to repeat?

Reply:
Thanks, thats a good suggestion. On reflection I think that it is ok to remain where it is as we don't repeat the same logic and it'd probably be harder code to read if we had an error_message variable created in a completely different context.